lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,59 @@
 local util = require 'lspconfig.util'
 
+local function activate_env(path)
```

Comment:
```suggestion
local function activate_env(path)
  assert(vim.fn.has('nvim-0.10') == 1, 'requires Nvim 0.10 or newer')
```

then, remove the `vim.loop` usages below, `get_active_clients`, and any other outdated apis.

Reply:
Removed support for <Neovim v0.10 in the command